substrate/primitives/core/Cargo.toml

```diff
@@ -94,6 +98,7 @@ std = [
 	"hash256-std-hasher/std",
 	"impl-serde/std",
 	"itertools",
+	"k256/std",
```

Comment:
We don't really need to enable `std`?

Reply:
thanks, leftover from prev iteration.
```suggestion
```

Reply:
hm..., looks like we need it, c[heck-features](https://github.com/paritytech/polkadot-sdk/actions/runs/8208792794/job/22452944449?pr=3525) is failing:
```
Run cargo featalign substrate/bin/node/runtime \
  cargo featalign substrate/bin/node/runtime \
    --features std,runtime-benchmarks,try-runtime \
    --mode check \
    --workspace-only \
    --ignore sc-executor \
    --default-std \
    --depth -1 | jq
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
{
  "sp-core 28.0.0 (path+file:///home/runner/work/polkadot-sdk/polkadot-sdk/substrate/primitives/core)": [
    {
      "id": "k256 0.13.3 (registry+https://github.com/rust-lang/crates.io-index)",
      "alias": "k256",
      "dependency-path": "/kitchensink-runtime/frame-benchmarking/frame-support/sp-api/sp-core",
      "problem": {
        "missing-features": [
          "std"
        ]
      }
    }
  ]
}
Error: Process completed with exit code 255
```